# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dump/dump-devfs.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./compat/include/pathnames.h.orig	2003-10-26 17:05:46.000000000 +0100
+++ ./compat/include/pathnames.h	2004-12-21 16:03:13.700942720 +0100
@@ -47,11 +47,11 @@
 
 #ifndef _PATH_DEFTAPE
 #ifdef __linux__
-#define	_PATH_DEFTAPE	"/dev/st0"
+#define	_PATH_DEFTAPE	"/dev/tape/tape0"
 #endif
 #ifdef sunos
 #define	_PATH_DEFTAPE	"/dev/rmt/0"
 #endif
 #endif
 
-#define	_PATH_RMT	"/etc/rmt"		/* path on remote host */
+#define	_PATH_RMT	"/sbin/rmt"		/* path on remote host */
